src/Tokenizer/Tokens.php

```diff
@@ -1139,6 +1157,26 @@ public function clearTokenAndMergeSurroundingWhitespace($index)
     }
 
     /**
+     * Finds the line number of the first token previous to the one at the specified index, including itself.
+     *
+     * @param int $index Token index
+     *
+     * @return int Line number
+     */
+    private function findNearestPrevTokenLineNumber($index)
+    {
+        if ($index <= 0) {
+            return 1;
```

Comment:
@SpacePossum It seems this code is not working correctly. This is supposed to ensure that if we hit the start of the file the line number must be line 1, but sometimes we receive a blank value, so something is not quite right here.